lisa/tools/lscpu.py

```diff
@@ -26,6 +29,18 @@ def _windows_tool(cls) -> Optional[Type[Tool]]:
     def _check_exists(self) -> bool:
         return True
 
+    def get_architecture(self, force_run: bool = False) -> str:
+        architecture: str = ""
+        result = self.run(force_run=force_run)
+        matched = self.__architecture_pattern.findall(result.stdout)
+        assert_that(
+            len(matched),
+            f"architecture should have exact one line, but got {matched}",
+        ).is_equal_to(1)
+        architecture = matched[0]
```

Comment:
We can have a validation, if it's a valid architecture string.

Reply:
where is the validation? Check if it's `x86_64` or other valid strings.

Reply:
need figure out what are the valid values.

Reply:
we can add it one by one. If it parses incorrect value, the error is easier to find.